jdaviz/core/astrowidgets_api.py

```diff
@@ -440,10 +441,17 @@ def add_markers(self, table, x_colname='x', y_colname='y',
                 raise AttributeError(f'{getattr(image, "label", None)} does not have a valid WCS')
             sky = table[skycoord_colname]
             t_glue = Data(marker_name, ra=sky.ra.deg, dec=sky.dec.deg)
+            dcomps = image.components
+            if isinstance(image.coords, gwcs.WCS) and 'Lon' in dcomps and 'Lat' in dcomps:
+                ra_str = 'Lon'
+                dec_str = 'Lat'

```

Comment:
is it guaranteed that these will always map to RA/Dec (and not galactic lon/lat, for instance)?

Reply:
No idea! Maybe @larrybradley can clarify or knows someone from Pipeline team who can.

Reply:
I'd prefer if we could get an answer on this before approving/merging... but if not, maybe we can just add a dev comment and deal with more complex logic later if necessary.

Reply:
Let's ping all the POs and see what happens: @larrybradley @orifox @camipacifici @PatrickOgle

Reply:
@pllim This will not work in general.  While the JWST will not produce a gwcs in the Galactic frame, presumably someone could make such a gwcs.  A gwcs object in the Galactic frame will also have "Lon"/"Lat".  Why not check the output coordinate frame directly from the gwcs (e.g., `wcsobj.output_frame.reference_frame.name`)?  @nden may have better ideas to generalize this.

 I'm not sure how glue uses these strings in linking to access the gwcs.

Reply:
Glue only cares about the high level API as defined in APE 14.

I'll see about updating the logic. Thanks!

Reply:
I'm confused why you need to manually set 'RA' and 'Dec'.  A `gwcs` object with the high-level API gives a SkyCoord the correct frame with RA/Dec:
```
>>> wcs.pixel_to_world(100, 100)
<SkyCoord (ICRS): (ra, dec) in deg
    (80.67259896, -69.45179937)>
```

Reply:
I need the component names to *link* the markers table to the image data. I need to know the names beforehand unless there is another way to grab the component names without caring whether it is RA/Dec or Lon/Lat in Glue?